src/plugins/vis_augmenter/public/ui_actions_bootstrap.ts

```diff
@@ -71,8 +71,15 @@ export const bootstrapUiActions = (uiActions: UiActionsSetup) => {
   uiActions.registerTrigger(externalActionTrigger);
   uiActions.registerTrigger(pluginResourceDeleteTrigger);
 
-  uiActions.addTriggerAction(EXTERNAL_ACTION_TRIGGER, openEventsFlyoutAction);
-  uiActions.addTriggerAction(CONTEXT_MENU_TRIGGER, viewEventsOptionAction);
-  uiActions.addTriggerAction(SAVED_OBJECT_DELETE_TRIGGER, savedObjectDeleteAction);
   uiActions.addTriggerAction(PLUGIN_RESOURCE_DELETE_TRIGGER, pluginResourceDeleteAction);
+  uiActions.addTriggerAction(EXTERNAL_ACTION_TRIGGER, openEventsFlyoutAction);
+
+  // These triggers are registered by other plugins. If they are disabled can throw an error.
+  try {
+    uiActions.addTriggerAction(CONTEXT_MENU_TRIGGER, viewEventsOptionAction);
+    uiActions.addTriggerAction(SAVED_OBJECT_DELETE_TRIGGER, savedObjectDeleteAction);
```

Comment:
wouldn't it be safer to wrap them individually? That way, a failure, due to one plugin being disabled, will not prevent other listeners from being added.

Reply:
I think I agree, but this may be fine for the current state of the augmenter feature. @ohltyler @lezzago , what's the intended behavior in the case were only management is disabled?

Reply:
Yeah with the current behaviour this is fine. The context menu trigger comes from the embeddables plugin which is not something we disable (Dashboards would not work without it). So in theory the only real action that cannot be registered is the savedObject Management trigger, which is already the second action in the try block.